Tang offline provisoning (#1544)
coreos/ignition: docs/configuration-v3_4_experimental.md

      * **_tang_** (list of objects): describes a tang server. Every server must have a unique `url`.
        * **url** (string): url of the tang server.
        * **thumbprint** (string): thumbprint of a trusted signing key.

>         * **advertisement** (string): A trusted advertisement json from a Tang server for initial provisioning. When supplied, the advertisement is used in place of fetching one from the Tang server on first boot. Thus enabling offline provisioning. 

Reviewer: The rest of this section uses `tang server`, so let's do that here too.

```suggestion
        * **advertisement** (string): the advertisement JSON. If not specified, the advertisement is fetched from the tang server during provisioning.
```
